docs/relnotes.md

```diff
@@ -17,6 +19,8 @@ Published on 2023-01.
 [`taipy-core` 2.1](https://pypi.org/project/taipy-core/2.1.0/) and
 [`taipy-rest` 2.1](https://pypi.org/project/taipy-rest/2.1.0/) packages.
 
+Please refer to [upgrades](./legacy-version-migration.md#from-2.0-to-2.1) page for
+details on how to migrate from version &#8804 2.0 to 2.1.
```

Comment:
```suggestion
details on how to migrate your legacy Taipy application from version &#8804 2.0 to 2.1.
```
The page is to help user migrate an application from 2.0 -> 2.1. If they start a new application, they would not need to read this.

Reply:
Hopefully... the page is for showing how to migrate from any version to the next, if necessary.
Just like the Release Notes, we don't want to have one page for each new version.

Reply:
Done.